test/ui-test/src/screens/StatusWelcomeScreen.py

```diff
@@ -222,24 +222,29 @@ def profile_settings_image_is_updated(self):
         # first time clicking on settings button closes the my profile modal
         click_obj_by_name(MainScreen.SETTINGS_BUTTON.value)
         click_obj_by_name(MainScreen.SETTINGS_BUTTON.value)
-        imagePresent("profiletestimage", True, 97, 100, 183, True)
+        myProfileSettingsObject = wait_and_get_obj(MainScreenComponents.PROFILE_SETTINGS_VIEW.value)
+        imagePresent("profiletestimage", True, 95, 100, 183, True, myProfileSettingsObject)
             
     def grab_screenshot(self):
         # take a screenshot of the profile image to compare it later with the main screen
         loginUserName = wait_and_get_obj(LoginView.LOGIN_VIEW_USER_IMAGE.value)
         grabScreenshot_and_save(loginUserName, "loginUserName", 200)
         
     def profile_image_is_updated(self):
-        profileNavBarButton = wait_and_get_obj(MainScreenComponents.PROFILE_NAVBAR_BUTTON.value)
-        imagePresent("loginUserName", True, 98, 85, 100, True)
-
+        # open profile popup and check image on profileNavBarButton and profileNavBarPopup
+        profileNavBarButton = wait_and_get_obj(MainScreenComponents.PROFILE_NAVBAR_BUTTON.value) #give it time to login
         click_obj(profileNavBarButton)
-        imagePresent("loginUserName", True, 99, 95, 100, True)
+        profilePopupImage = wait_and_get_obj(ProfilePopup.USER_IMAGE.value)
+        imagePresent("loginUserName", True, 95, 75, 100, True, profileNavBarButton)
+        imagePresent("loginUserName", True, 95, 75, 100, True, profilePopupImage)
         
     def enter_password(self, password):
         click_obj_by_name(LoginView.PASSWORD_INPUT.value)
         type(LoginView.PASSWORD_INPUT.value, password)
         click_obj_by_name(LoginView.SUBMIT_BTN.value)   
+    
+    def waitForMainWindowAfterLogin(self):
```

Comment:
We already have this validation in a different step so I suggest to remove this method and instead add the following step in the corresponding scenario: 
`When the user logs in with password "TesTEr16843/!@00"`
--> Add this: `And the user lands on the signed in app` --> It makes the validation after the login

Reply:
Cool, didn't notice this step in the test files. Done